department-of-veterans-affairs/caseflow
# Added message for IE users to use a different browser

Resolves [https://vajira.max.gov/browse/CASEFLOW-1995](https://vajira.max.gov/browse/CASEFLOW-1995)

### Description

Adds a help message letting IE users know that their browser is not supported and providing them an option to use a different browser.

### Acceptance Criteria
- [ ] IE users receive a message telling them to use a different browser
- [ ] Edge/Chrome users can still access Caseflow

### Testing Plan
1. Follow the setup instructions for the [JAWS Testing](https://github.com/department-of-veterans-affairs/appeals-team/blob/master/Engineering/accessibility-jaws-setup.md) so that you can get a windows machine running and connected to your local caseflow instance
2. Open Internet Explorer 11
3. Navigate to http://10.0.2.2:3000/test/users
4. Ensure you receive a message indicating that your browser is unsupported
5. Test the Edge link and ensure that the link opens in Microsoft Edge
6. Test the Chrome link and ensure it takes you to download chrome (there is not a way to open the app in chrome using a link at the moment so this seemed to be the best option)
